test/smoke/testApps/SimpleEventData/src/main/java/com/microsoft/ajl/simplecalc/SimpleCalculatorServlet.java

```diff
@@ -0,0 +1,119 @@
+package com.microsoft.ajl.simplecalc;
+
+import com.microsoft.ajl.simplecalc.model.BinaryCalculation;
+import com.microsoft.ajl.simplecalc.model.BinaryOperator;
+import com.microsoft.applicationinsights.TelemetryClient;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static com.microsoft.ajl.simplecalc.ParameterConstants.*;
+
+/**
+ * Servlet implementation class SimpleCalculatorServlet
+ */
+@WebServlet(description = "Performs given calculation", urlPatterns = { "/doCalc" })
+public class SimpleCalculatorServlet extends HttpServlet {
+	private static final long serialVersionUID = -633683109556605395L;
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		BinaryCalculation bc = null;
+		try {
+			bc = readParameters(request.getParameterMap());
+		}
+		catch (CalculatorParameterException cpe) {
+			String errMsg = cpe.getLocalizedMessage();
+			System.err.println(errMsg);
+			response.sendError(HttpServletResponse.SC_BAD_REQUEST, errMsg);
+			return;			
+		}
+		
+		if (bc == null) {
+			System.out.println("No parameters given.");
+			response.setStatus(HttpServletResponse.SC_NO_CONTENT);
+			return;
+		}
+		
+		response.setContentType("text/html;charset=UTF-8");
+		renderHtml(bc, response.getWriter());
+		
+		new TelemetryClient().trackEvent("EventDataTest");
+
+		Map<String, String> properties = new ConcurrentHashMap<String, String>();
```

Comment:
<> types can auto infer. Please do not provide explicit types. explained in one of the comments above.

Reply:
Thank you very much, I have changed the code as you advise :
 Map properties = new ConcurrentHashMap();

Reply:
@Terryxxx I do not see the changes in the file yet. Have you checked in the code? I meant it in this way:
Map<String, String> properties = new ConcrurrentHashMap<>(); 
Please make sure it is done like this. Also none of the changes above are yet seen in the code. Collaborate with @littleaj to make sure things are at right place.

Reply:
I may have started the project in java 1.6 compatibility mode and changed it later. This is very minor and I wouldn't worry about it.

Reply:
@dhaval24 Sorry, I didn't check code yesterday. But I'm checking the code now.